index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/ea73301412.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <!-- Home -->
+    <section class="home bg-home">
+        <!-- HEADER -->
+        <header>
+            <div class="top-bar">
+                <ul class="top-bar-list">
+                    <li class="topbar-item"><a href=""><i class="fa-brands fa-facebook-f"></i></a></i></li>
+                    <li class="topbar-item"><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li class="topbar-item"><a href="">English</a></li>
+                    <li class="topbar-item"><a href="">My Page</a></li>
+                    <li class="topbar-item"><a href="">Logout</a></li>
+                </ul>
+            </div>
+            <nav class="menu">
+                <button class="hamb-icon">
+                    <i class="fa-sharp fa-solid fa-bars"></i>
+                </button>
+
+                
+                <ul class="navbar">
+                    <button class="close-btn"><i class="fa-solid fa-xmark"></i></button>
+                    <li class="nav-item"><a href="about.html">About</a></li>
+                    <li class="nav-item"><a href="">Activities</a></li>
+                    <li class="nav-item"><a href="">Join</a></li>
+                    <li class="nav-item"><a href="">Sponsor</a></li>
+                    <li class="nav-item"><a href="">How to get</a></li>
+                    <li class="nav-item"><a class="sea-event" >Season Events</a></li>
+                </ul>
+
+                <a href="">
+                    <img class="logo" src="Images/Home/Logo_Cafayate.png" alt="">
+                </a>
+            </nav>
+
+        </header>
+        <!-- Main -->
+        <main>
+            <div class="main-container">
+                <div class="heading">
+                    <h3>Sun & good wine land</h3>
+                    <h1>Cafayate</h1>
+                    
+                </div>
+                <div class="par-date" >
+                    <article>
+                        Cafayate is a town in the Calchaquíes valleys located in the southwest of the province of Salta, northwestern region of Argentina. It is the head of the department of Cafayate and recognized for the quality of the wines produced there.
+                    </article>
+                    <div class="date-place">
+                        <h2>2023.9.6(WED) ~ 8(FRI)</h2>
+                        <p>@National Technological University</p>
+                    </div>
+                </div>
+                
+            </div>
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement according to which you can make little changes or adjustments of your own choice but can't change it completely but as I can see that your design looks different than actual, which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://github.com/ulises2607/cayafate-project/assets/66476329/cd029716-65ab-45b5-8e7d-6886f05d48af)


### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-22 16-21-56](https://github.com/ulises2607/cayafate-project/assets/66476329/e71bdc71-6a09-48bd-a21a-2f6f722cbb08)

</td>
<td>

![image](https://github.com/ulises2607/cayafate-project/assets/66476329/cf02ea77-62f4-4754-b31e-d066ef97f3ef)

</td>
</tr>
</table>

### Mobile Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/ulises2607/cayafate-project/assets/66476329/b517f24f-a8a5-4845-b752-9cf3d77eb8c2)

</td>
<td>

![Screenshot from 2023-07-08 15-38-44](https://github.com/ulises2607/cayafate-project/assets/66476329/41667559-a4fd-4b14-8b4c-179408227bed)

</td>
</tr>
</table>